core/src/main/java/io/grpc/ExponentialBackoffPolicy.java

```diff
@@ -0,0 +1,40 @@
+package io.grpc;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import java.util.Random;
+
+/**
+ * Retry Policy for Transport reconnection.
+ */
+class ExponentialBackoffPolicy {
```

Comment:
@carl-mastrangelo I still think it's worth-while to just define a `RetryPolicy` interface with a couple of methods that we need. WDYT?